website/content/docs/concepts/identity.mdx

```diff
@@ -20,11 +20,13 @@ Each user may have multiple accounts with various identity providers, and Vault
 supports many of those providers to authenticate with Vault. Vault Identity can
 tie authentications from various auth methods to a single representation. This representation of a consolidated identity is called an **Entity** and their
 corresponding accounts with authentication providers can be mapped as
-**Aliases**. In essence, each entity is made up of zero or more aliases. 
+**Aliases**. In essence, each entity is made up of zero or more aliases. An entity cannot have more than one alias for
+a particular authentication backend.
 
 For example, a user with accounts in both GitHub and LDAP can be mapped to a
 single entity in Vault with two aliases, one of type GitHub and one of type
-LDAP.
+LDAP. Note however, if both aliases are created on the same auth mount, say
```

Comment:
This file has different content in `main` (more words and extra images) - is it worth getting those into the release branch?

Reply:
I'm not sure if those changes should be backported or whether they are already inflight as backport PRs elsewhere. I've kept the changes from other files strictly the same as what's in https://github.com/hashicorp/vault/pull/13370/files.